wd_capabilities.au3

```diff
@@ -384,8 +364,6 @@ Func __WD_CapabilitiesInitialize($s_MatchType, $s_BrowserName = '')
 		$s_SpecificOptions_KeyName = ''
 	ElseIf $s_MatchType = 'firstMatch' And $s_BrowserName = '' Then
 		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_NotSupported, "FirstMatch requires BrowserName to be defined"))
-;~ 	Else
```

Comment:
this else is not needed as `$_WD_KEYS__MATCHTYPES` check it before calling `__WD_CapabilitiesInitialize`